src/domains/edit/tests/add-wheelchair-description.e2e-spec.ts

```diff
@@ -0,0 +1,74 @@
+import { type Locator, expect, test } from "@playwright/test";
+import node4544823443Mock from "~/domains/edit/tests/mocks/node-4544823443-osm-mock.json";
+import {
+  getButton,
+  getDialog,
+  getEditButton,
+  getMenuItem,
+  selectLanguage,
+  setupPage,
+} from "~/domains/edit/tests/utils";
+
+test.describe("Add wheelchair description in new language", () => {
+  let dialog: Locator;
+
+  test.beforeEach(async ({ page }) => {
+    await setupPage(page);
+    await getEditButton(page, "wheelchair:description").click();
+    await getMenuItem(page, "new-language").click();
+    dialog = await getDialog(page);
+  });
+
+  test("dialog is rendered", async () => {
+    await expect(dialog).toBeVisible();
+  });
+
+  test("dialog has select box when adding a new language", async () => {
+    await expect(dialog.getByRole("combobox")).toBeVisible();
+  });
+
+  test("dialog content is key board navigable", async () => {
+    //TODO
+  });
+
+  test("text area is hidden before language is selected", async ({ page }) => {
+    const textArea = dialog.getByRole("textbox");
+    await expect(textArea).toBeHidden();
+
+    await selectLanguage(page, dialog, "English");
```

Comment:
praise: I really like this. It makes it very easy to understand, what the test is doing. Great job! 👍